### PR TITLE
Fix command readme

### PR DESCRIPTION
**Pull request by ceritium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ You can deploy this application anywhere you can run a Rack app. (Azure, Heroku,
 1. Download the repo `git clone https://github.com/substrakt/letsencrypt-heroku.git`
 1. Install Redis. (`brew install redis`)
 1. Install foreman (`gem install foreman`)
-1. Copy `.envsample` to `.env` using `cp .envsample .env`. The `.env` file is read when the application starts and should contain all of the required environment variables. One of these is the token generated earlier for Heroku. **DO NOT COMMIT THIS FILE TO SOURCE CONTROL**
+1. Copy `.envsample` to `.env` using `cp .env.sample .env`. The `.env` file is read when the application starts and should contain all of the required environment variables. One of these is the token generated earlier for Heroku. **DO NOT COMMIT THIS FILE TO SOURCE CONTROL**
```
Comment (maxehmookau) [inline README.md]: This is better but can you update the first reference to `.envsample` too please?

**Review by maxehmookau — CHANGES_REQUESTED** [review]

**Comment by ceritium** [comment]
I am going to merge que commits.

**Comment by ceritium** [comment]
Ready

**Review by maxehmookau — APPROVED** [review]

**Comment by maxehmookau** [comment]
Great, thank you!

**Pull request merged**